content/en/continuous_integration/pipelines/_index.md

```diff
@@ -155,21 +155,22 @@ If your CI provider is not supported, you can try setting up Pipeline Visibility
+| {{< ci-details title="Custom spans for traced commands using datadog-ci" >}}Support for sending command-level events to CI Visibility to be incorporated into pipeline flame graph visualization. You can then query and analyze <a href="https://docs.datadoghq.com/continuous_integration/pipelines/custom_commands/">these events</a>. {{< /ci-details >}}                                    |                  {{< X >}}                   |           | {{< X >}} |           |   {{< X >}}    |                 |           |           |                   |                    |
+| {{< ci-details title="Custom predefined tags" >}}Support for setting static pipeline tags in the CI provider that do not change between executions.{{< /ci-details >}}                                                                                                                                                                                                                           |                  {{< X >}}                   | {{< X >}} | {{< X >}} |           |   {{< X >}}    |    {{< X >}}    |           |           |                   |                    |
+| {{< ci-details title="Custom tags and metrics at runtime" >}}Support for adding <a href="https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/">user-defined text and numerical tags</a> to pipelines and jobs in CI Visibility.{{< /ci-details >}}                                                                                                               |                  {{< X >}}                   | {{< X >}} | {{< X >}} | {{< X >}} |                |    {{< X >}}    |           |           |                   |     {{< X >}}      |
+| {{< ci-details title="Parameters" >}}Support for adding custom pipeline parameters that users set (for example, <code>DYNAMICS_IS_CHILD:true</code>). You can then search using these parameters in the <a href="https://docs.datadoghq.com/continuous_integration/explorer/?tab=pipelineexecutions">CI Visibility Explorer</a> to find all events with a specific parameter.{{< /ci-details >}} |                  {{< X >}}                   | {{< X >}} |           |           |                |                 | {{< X >}} |           |                   |     {{< X >}}      |
+| {{< ci-details title="Pipeline failure reason" >}}Identification of a specific reason behind a pipeline or job failure.{{< /ci-details >}}                                                                                                                                                                                                                                                       |                  {{< X >}}                   | {{< X >}} |           |           |                |                 | {{< X >}} | {{< X >}} |     {{< X >}}     |     {{< X >}}      |
 
```

Comment:
Let's keep the same syntax as before? I would expect just a {{< X >}} change here

Reply:
oh crap my IDE did some nasty things